clang/test/CodeGen/inline-asm-sizeless.c

```diff
@@ -0,0 +1,7 @@
+// RUN: not %clang_cc1 -S %s -o /dev/null 2>&1 | FileCheck %s
```

Comment:
Please use "-verify" to check clang diagnostics, not FileCheck.

Is there any existing file with inline asm diagnostics this can be added to?